gandalf.js

```diff
@@ -1,44 +1,53 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!